src/element/textWysiwyg.tsx

```diff
@@ -317,10 +321,17 @@ export const textWysiwyg = ({
         editable.style.height = height;
         editable.style.height = `${editable.scrollHeight}px`;
       }
+
       onChange(normalizeText(editable.value));
     };
   }
 
+  const handleSelectionChange = () => {
+    onSelection({ start: editable.selectionStart, end: editable.selectionEnd });
+  };
+
+  document.addEventListener("selectionchange", handleSelectionChange);
```

Comment:
Tried a few ways to subscribe to selections in a text area but this seemed to be the only method that worked reliably. It's a bit annoying to have to register a global listener tho.